make/langtools/tools/javacserver/client/Client.java

```diff
@@ -51,10 +51,10 @@
 public class Client {
     private static final Log.Level LOG_LEVEL = Log.Level.INFO;
 
-    // Wait 2 seconds for response, before giving up on javac server.
-    private static final int CONNECTION_TIMEOUT = 2000;
-    private static final int MAX_CONNECT_ATTEMPTS = 3;
-    private static final int WAIT_BETWEEN_CONNECT_ATTEMPTS = 2000;
+    // Wait 4 seconds for response, before giving up on javac server.
+    private static final int CONNECTION_TIMEOUT = 4000;
+    private static final int MAX_CONNECT_ATTEMPTS = 10;
+    private static final int WAIT_BETWEEN_CONNECT_ATTEMPTS = 4000;
```

Comment:
Not sure increasing the wait time is a good idea. If I'm reading this right, that means 8 seconds between each attempt if they are timing out. Will your problem still be alleviated if we keep the wait time at 2000?